server/src/controller/authentication.ts

```diff
@@ -0,0 +1,65 @@
+import express from "express";
+import jwt from "jsonwebtoken";
+import { authentication, random } from "../helpers/index";
+import { UserService } from "../service/user";
+import { User } from "models/user";
+
+export class AuthController {
+  static async register(req: express.Request, res: express.Response) {
+    try {
+      const { email, password, firstname, lastname } = req.body;
+
```

Comment:
Can you add a validation that checks if the body contains these params, if no throw an error with bad request?